analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -75,8 +75,7 @@ class Gradle(
     name: String,
     analysisRoot: File,
     analyzerConfig: AnalyzerConfiguration,
-    repoConfig: RepositoryConfiguration,
-    private val gradleVersion: String? = null
+    repoConfig: RepositoryConfiguration
```

Comment:
commit message: "This allows the Gradle version to also be
configured when creating the Gradle instance via the factory class."

It'd be nice to have a hint why this can be useful.